spec/duration.html

```diff
@@ -1289,27 +1289,33 @@ <h1>RoundDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _sec
           1. Let _monthsWeeksInDays_ be ? DaysUntil(_yearsLater_, _yearsMonthsWeeksLater_).
           1. Set _relativeTo_ to _yearsLater_.
           1. Let _days_ be _days_ + _monthsWeeksInDays_..
+          1. Let _daysDuration_ be ? CreateTemporalDuration(0, 0, 0, _days_, 0, 0, 0, 0, 0, 0).
+          1. Let _daysLater_ be ? Call(_dateAdd_, _calendar_, « _relativeTo_, _daysDuration_, _options_, %Temporal.PlainDate% »).
+          1. Let _untilOptions_ be ! OrdinaryObjectCreate(%Object.prototype%).
+          1. Perform ! CreateDataPropertyOrThrow(_untilOptions_, *"largestUnit"*, *"years"*).
+          1. Let _dateUntil_ be ? GetMethod(_calendar_, *"dateUntil"*).
+          1. Let _timePassed_ be ? Call(_dateUntil_, _calendar_, « _relativeTo_, _daysLater_, _untilOptions_ »).
```

Comment:
Another rebase for #1393

Reply:
These actually don't have to be rebased. There were already CalendarDateAdd and CalendarDateUntil abstract operations existing in the spec text, so you can just change this to use them regardless of when 1393 is merged.

Reply:
done